akka-http-marshallers-java/akka-http-jackson/src/main/scala/akka/http/javadsl/marshallers/jackson/Jackson.scala

```diff
@@ -23,16 +26,16 @@ object Jackson {
         unmarshalling.Unmarshaller.stringUnmarshaller
           .forContentTypes(`application/json`)
           .map { jsonString ⇒
-            val reader = new ObjectMapper().reader(clazz)
+            val reader = objectMapper.reader(clazz)
             clazz.cast(reader.readValue(jsonString))
           }
       }
     }(ClassTag(clazz))
 
-  private val _jsonMarshaller: Marshaller[AnyRef] =
+  private def jsonMarshaller(objectMapper: ObjectMapper): Marshaller[AnyRef] =
     MarshallerImpl[AnyRef] { implicit ec ⇒
       marshalling.Marshaller.StringMarshaller.wrap(`application/json`) { (value: AnyRef) ⇒
-        val writer = new ObjectMapper().enable(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY).writer()
+        val writer = objectMapper.enable(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY).writer()
```

Comment:
We cannot change possibly shared setting here. Let's do it for the predefined ObjectMapper during initialization.